habitat_baselines/utils/gym_definitions.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import os.path as osp
+from typing import Any, List
+
+from gym.envs.registration import register
+
+import habitat
+import habitat_baselines.utils.env_utils
+from habitat_baselines.common.environments import get_env_class
+from habitat_baselines.config.default import _C
+from habitat_baselines.config.default import get_config as baselines_get_config
+from habitat_baselines.utils.gym_adapter import HabGymWrapper
+from habitat_baselines.utils.render_wrapper import HabRenderWrapper
+
+GYM_AUTO_NAME_KEY = "GYM_AUTO_NAME"
+HABLAB_INSTALL_PATH = "HABLAB_BASE_CFG_PATH"
+
+if HABLAB_INSTALL_PATH in os.environ:
+    base_dir = os.environ[HABLAB_INSTALL_PATH]
+else:
+    base_dir = osp.dirname(osp.dirname(osp.dirname(osp.abspath(__file__))))
+
+
+def get_config_no_base_task_load(cfg_file_path):
```

Comment:
Are these two methods public facing (do we expect users to know about them and use them)? If yes, I think the comments should explain them a little more, other wise, I think these should be have a `_` prefix.

Reply:
They are indeed private methods, fixed.